templates/security/cve/_cve-landing.html

```diff
@@ -0,0 +1,51 @@
+<section class="p-section">
+  <div class="row--25-75">
+    <hr class="p-rule"/>
+    <div class="col">
+        <h2>Search CVEs</h2>
+    </div>
+    <div class="col">
+      {% include "security/cve/_cve-search.html" %}
+    </div>
+  </div>
+</section>
+
+<section class="p-section cve-landing">
+  <div class="row">
+    <div class="col-start-large-4 col-9">
+      <hr class="p-rule"/>
+      <div class="row">
+        <div class="col-2">
+          <h3 class="p-heading--5">By Ubuntu release</h3>
+        </div>
+        <div class="col-7">
+          <ul class="p-inline-list">
+            <li class="p-inline-list__item"><a href="/security/cves?{% for release in maintained_releases %}version={{release.codename}}&status=&{% endfor %}">All maintained releases</a></li>
+            <li class="p-inline-list__item"><a href="/security/cves?{% for release in lts_releases %}version={{release.codename}}&status=&{% endfor %}">All LTS</a></li>
```

Comment:
This is the only line that was actually changed here, rest is just this section being moved to a partial

Reply:
Let's say the `lts_releases = ['14.04', '16.04', '18.04']`
That means that your link's href will look like this:
```
/security/cves?version=14.04&status=&version=16.04&status=&version=18.04&status=&
```
This doesn't look right.

Reply:
Status isn't needed anymore, you're right (used to be for the cve table, but we're not bound by that anymore). Correct href is `/security/cves?version=precise&version=lucid&version=hardy&version=dapper`